libs/langchain/langchain/document_loaders/directory.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         self,
         path: str,
         glob: str = "**/[!.]*",
+        exclude_glob: Optional[str] = None,
```

Comment:
maybe i don't fully understand, but should doesn't glob already have enough expressivity to exclude whatever you'd like

Reply:
I don't think so, there doesn't seem to be a way to exclude anything complicated at least (for example: include all *.py files except the ones that start with test_, or, like a say here, include all subdirectories except one). This answer on SO https://stackoverflow.com/questions/20638040/glob-exclude-pattern suggests to do the exclusion in Python.